EstateAdministrationUI/Areas/Estate/Controllers/HomeController.cs

```diff
@@ -75,6 +75,47 @@ public async Task<IActionResult> CreateMerchant(CancellationToken cancellationTo
             return this.View("CreateMerchant", viewModel);
         }
 
+        /// <summary>
+        /// Creates the merchant.
+        /// </summary>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        /// <returns></returns>
+        [HttpGet]
+        public async Task<IActionResult> CreateOperator(CancellationToken cancellationToken)
+        {
+            CreateOperatorViewModel viewModel = new CreateOperatorViewModel();
+
+            return this.View("CreateOperator", viewModel);
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> CreateOperator(CreateOperatorViewModel viewModel,
+                                                        CancellationToken cancellationToken)
+        {
+            // Validate the model
+            if (this.ValidateModel(viewModel))
+            {
+                String accessToken = await this.HttpContext.GetTokenAsync("access_token");
+
+                CreateOperatorModel createOperatorModel = this.ViewModelFactory.ConvertFrom(viewModel);
+
+                // All good with model, call the client to create the operator
+                var createOperatorResponse =
```

Comment:
Codacy found an issue: [Remove the unused local variable 'createOperatorResponse'.](https://app.codacy.com/manual/stuart_ferguson1/EstateAdministrationUI/pullRequest?prid=6044972)